test/unit/dropdown_spec.js

```diff
@@ -75,6 +75,34 @@ describe('Dropdown', function() {
     });
   });
 
+  describe('when instantiated with a custom empty template', function() {
+    beforeEach(function() {
+      this.view.destroy();
+      this.view = new Dropdown({
+        menu: this.$menu,
+        datasets: [{}],
+        templates: {
+          empty: '<h3 class="empty">this is empty</3>'
```

Comment:
Won't you end up with a `<div class="empty"><h3 class="empty">this is empty</h3></div>` here (two `empty` classes)?